js/src/search/components/SearchVisualizer.vue

```diff
@@ -3,18 +3,30 @@
     <GraphVisualizerBase :graph="graph" :transitions="true" :layout="layout" :legendColor="legendColor" :legendText="legendText">
       <template slot="node" scope="props">
         <RoundedRectangleGraphNode :id="props.node.id" hover="props.hover" :text="props.node.name" :textColour="nodeTextColour(props.node, props.hover)"
-                          :subtext="showNodeHeuristics ? nodeHText(props.node) : undefined" :simplifyGraph="simplifyGraph"
-                          :fill="nodeFillColour(props.node, props.hover)" :hover="props.hover"
-                          :stroke="nodeStroke(props.node)" :stroke-width="nodeStrokeWidth(props.node)"
-                          @updateBounds="updateNodeBounds(props.node, $event)" :textSize="textSize">
+                                   :subtext="showNodeHeuristics ? nodeHText(props.node) : undefined" :simplifyGraph="simplifyGraph"
+                                   :fill="nodeFillColour(props.node, props.hover)" :hover="props.hover"
+                                   :stroke="nodeStroke(props.node)" :stroke-width="nodeStrokeWidth(props.node)"
+                                   @updateBounds="updateNodeBounds(props.node, $event)" :textSize="textSize">
         </RoundedRectangleGraphNode>
       </template>
       <template slot="edge" scope="props">
         <DirectedRectEdge :id="props.edge.id" :x1="props.edge.source.x" :x2="props.edge.target.x" :y1="props.edge.source.y" :y2="props.edge.target.y" :stroke="props.edge.styles.stroke"
-                      :strokeWidth="props.edge.styles.strokeWidth" :text="edgeText(props.edge)" :nodeName="props.edge.target.name"
-                      :graph_node_width="props.edge.styles.targetWidth" :graph_node_height="props.edge.styles.targetHeight">
+                          :strokeWidth="props.edge.styles.strokeWidth" :text="edgeText(props.edge)" :nodeName="props.edge.target.name"
+                          :graph_node_width="props.edge.styles.targetWidth" :graph_node_height="props.edge.styles.targetHeight">
         </DirectedRectEdge>
       </template>
+      <template slot="visualization" scope="props">
+        <foreignObject class="dropdown" :x="btnProp(props.width).x" :y="btnProp().y" width="100%">
+          <button class="dropbtn">Visualization Options</button>
+          <div class="dropdown-content">
+            <a @click="$emit('toggle:showFullDomain')">Switch Graph</a>
```

Comment:
I think Show Full Domain would be a better name for the button than Switch Graph.